docusaurus/android_versioned_docs/version-draft/01-basics/01-overview.mdx

```diff
@@ -0,0 +1,140 @@
+---
+slug: /
+---
+
+# Overview
+
+Built on top of the Stream Chat low-level client, the Stream Chat Android UI components enable you to easily build any type of chat or messaging experience for Android.
+
+We have UI component libraries available for both Android Views and Jetpack Compose. Each of these libraries provides an extensive collection of efficient and customizable UI components which enable you to quickly get started with little to no setup required. The libraries support:
+
+<!-- TODO validate list below. It's a copy from iOS, should we add/remove something? -->
+- Rich-media messages
+- Channel and message lists
+- Message Reactions
+- Message threads and quoted replies
+- Text input commands (ex: Giphy and @mentions)
+- Image and file uploads
+- Video playback
+- Indicators for read state and typing
+- Push notifications
+- Offline storage
+- Voice messages
+- and more.
+
+:::note
+The fastest way to get started with the SDK is by trying the [Android Views In-App Messaging Tutorial](https://getstream.io/tutorials/android-chat/). If you're using Jetpack Compose, see the [Compose In-App Messaging Tutorial](https://getstream.io/chat/compose/tutorial/) instead.
+:::
+
+## Architecture
+
+The SDK consists of three separate libraries:
+<!-- TODO iOS uses the artifact name here, for us it would be stream-chat-android-ui-components etc -->
+<!-- TODO Name should be "UI Components" or "Android Views/XML UI Components"? -->
+<!-- TODO Should we list Compose components first? -->
```

Comment:
TODO

Reply:
We could.
Does iOS list components for `UIKit` and `SwiftUI` separately?